### DISCUSSION TITLE
Pixieset like (beautiful) Album Sharing

### DISCUSSION THREAD

**Discussion by moritzbindewald** [immich-app/immich#10443] [Feature Request]
Hey there, I love Immich immensely. Thx so much to Alex and the Team! 

Now to my "request": I recently discovered Pixieset and it has a beautiful way to share gallerys for friends/clients. I would love to see Immich take inspiration from the folkes over there. Pictures immediately just feel so much more "valuable" if they are in a package as beautiful as this. 

I understand, that Pixieset-like album creation and sharing is not as simple as clicking share and just creating a link (By that I mean the way immich does it). 

Maybe one could simplify the process by allowing to set a custom Album thumbnail? (If I am correct, that is not possible on mobile yet) 

I also find the option for public users to <3 (like) shared images and only download those quite neat. 

This would be so great for my purposes (hobby/professional photographer). 

Thx in advance for considering/debating the idea!!!

And again: thx for developing such a wonderful peace of open source software 😀 

![image](https://github.com/immich-app/immich/assets/106192865/23498950-7ad4-44b4-bd02-6928d45ee63b)

![image](https://github.com/immich-app/immich/assets/106192865/2ded3e6a-b8e7-4bac-9f1c-ec9a5a9bff6d)

 ![image](https://github.com/immich-app/immich/assets/106192865/e5e32ec2-2079-4944-aaf1-da463ee1a810)

**Comment by Zoldo** [comment]
Since I need this functionality as well (or at least part of it), I also hope that this feature can be implemented. 
Since it includes multiple sub-features that can be implemented separately, I opened another feature request with just the favouriting of images: #11642
